content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/serial_numbers.rst

```diff
@@ -1,147 +1,230 @@
-system will automatically identify the current location of the product,
-based on its last movement.
+*Serial Numbers* are one of the two ways to identify and track products in Odoo. A serial number
+is a unique identifier assigned incrementally (or sequentially) to an item or product, so as to
+identify it in order to distinguish it from other items and products.
 
-If you want to do it or run very advanced traceability, the Odoo
-double-entry management is the feature you need.
+Serial numbers can consist of many different types of characters: they can be strictly numerical;
+they can contain letters and other typographical symbols; and they might even consist entirely of
+character string. The goal of assigning serial numbers to individual products is to make sure that
+every item's history is identifiable when it travels through the supply chain. This can be
+especially useful for manufacturers that provide after-sales services to products that they sell
+and deliver.
 
-Configurations
-==============
+.. seealso::
+    - :doc:`lots`
 
-Application configuration
--------------------------
+Configuration
+=============
 
-First, you need to activate the serial numbers tracking. To do so, go to
-:menuselection:`Configuration --> Settings` in the *Inventory* application. Then,
-enable the feature and hit save.
+To track products using *Serial Numbers*, the :guilabel:`Lots & Serial Numbers` feature must be
+enabled. To enable this, go to :menuselection:`Inventory --> Configuration --> Settings`, scroll
+down to the :guilabel:`Traceability` section, and click the box next to
+:guilabel:`Lots & Serial Numbers`. :guilabel:`Save` changes.
 
-.. image:: serial_numbers/serial_numbers_01.png
-    :align: center
+.. image:: serial_numbers/serial-numbers-enabled-setting.png
+   :align: center
+   :alt: Enabled lots and serial numbers setting.
 
-Product configuration
----------------------
+Track products by serial number
+===============================
 
-Now, you have to configure the products you want to track by serial
-numbers. To do so, go to :menuselection:`Master Data --> Products` and open the product
-of your choice. Edit it and select *Tracking By Unique Serial Number*
-in the *Inventory tab*. Then, click on save.
+Once the :guilabel:`Lots & Serial Numbers` setting has been activated, individual products can now
+be configured to be tracked using serial numbers. To do this, go to :menuselection:`Inventory -->
+Products --> Products`, and choose a product. Once on the product form, click the
+:guilabel:`Inventory` tab.
 
-.. image:: serial_numbers/serial_numbers_02.png
-    :align: center
+To track this product using serial numbers, click :guilabel:`Edit`, and under the
+:guilabel:`Traceability` section, click :guilabel:`By Unique Serial Number`. :guilabel:`Save`
+changes. Existing or new serial numbers can now be assigned to newly-received or manufactured
+batches of this product.
```

Comment:
Will serial numbers be automatically assigned to newly created products if they have "by unique serial number" enabled?